### PR TITLE
fix td style

### LINKED ISSUE #243: TD style prop is not passed down to children
The "style" prop from a Reactable.Td doesn't pass down to children.  It seems to work fine with Reactable.Tr.

### PR DESCRIPTION
**Pull request by y2468101216**
close #243

### CODE REVIEW

File: src/reactable/td.jsx [hunk 1]
```diff
@@ -15,6 +15,15 @@ export class Td extends React.Component {
             className: this.props.className,
             onClick: this.handleClick.bind(this)
         };
+        
+        if(typeof(this.props.style) !== 'undefined'){
+            try{
+                tdProps.style = JSON.parse(this.props.style);
```
Comment (glittershark) [inline src/reactable/td.jsx]: Why not just pass it in as an object?

**Comment by y2468101216** [comment]
you are right, sorry for my stupid.

File: src/reactable/td.jsx (hunk 1 above)
Comment (y2468101216) [inline src/reactable/td.jsx]: you are right, sorry for my stupid.

File: build/reactable.js [hunk 2]
```diff
@@ -443,6 +443,10 @@ window.ReactDOM["default"] = window.ReactDOM;
                     onClick: this.handleClick.bind(this)
                 };
 
+                if (typeof this.props.style !== 'undefined') {
```
Comment (glittershark) [inline build/reactable.js]: I've used typeof as `typeof(this.props.style)` elsewhere in the codebase, let's keep that consistent

File: tests/reactable_test.jsx [hunk 3]
```diff
@@ -494,6 +494,31 @@ describe('Reactable', function() {
...
+            });
+
+            after(ReactableTestUtils.resetTestEnvironment);
+
+            it('renders the first column with the width', function() {
+                expect($('td.name-1')).to.have.attr('style').match(/width/);
+            });
+
+          
+
```
Comment (glittershark) [inline tests/reactable_test.jsx]: looks like this block is over-indented, plus get rid of these three blank lines

**Comment by glittershark** [comment]
Fix that minor code style nit and squash everything to one commit and this is good to go

**Pull request closed without merging**